ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/integrations.tsx

```diff
@@ -94,6 +95,21 @@ const INTEGRATIONS: Integration[] = [
     description:
       'Check out our docs to see how you can use Inngest with your applications deployed to Netlify.',
   },
+  {
+    title: 'Prometheus',
+    slug: 'prometheus',
+    Icon: <IconPrometheus className="text-onContrast h-6 w-6" />,
+    actionButton: () => (
+      <Button
+        iconSide="left"
+        appearance="solid"
+        size="medium"
+        label="Configure"
+        href="/settings/integrations/prometheus"
+      />
+    ),
+    description: 'Configure your Prometheus instance to scrape metrics directly from Inngest.',
+  },
```

Comment:
Can we add a feature flip for this similar to how the supabase one is done on this file? We then can enable it for enterprise folks while we work on docs/blog post/changelog update?

OR we could just pre-approve this PR if we're confident that it works and just merge this when we want to make this feature live this week. What do you think @cdzombak ?